corehq/ex-submodules/casexml/apps/phone/tests/utils.py

```diff
@@ -72,15 +73,27 @@ def generate_restore_payload(project, user, restore_id="", version=V1, state_has
         returns: the xml payload of the sync operation
     """
     return get_restore_config(
-        project, user, restore_id, version, state_hash, items, overwrite_cache, force_cache
+        project, user, restore_id, version, state_hash, items, overwrite_cache,
+        force_cache, **kw
     ).get_payload().as_string()
 
 
+def get_next_sync_log(*args, **kw):
```

Comment:
https://github.com/dimagi/commcare-hq/blob/972114929ecdb85b4de4971eea341d6f8925e41e/corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py#L187 - github is not loading the diff for the file where it's used, but you can manually expand it.